sink/sink

```diff
@@ -118,29 +158,62 @@ class GitHub(object):
                                response.status, response.reason, output.strip()))
         return json.loads(output)
 
+    def cleanup(self):
+        child = self.child
+        self.child = 0
+        if child == 0:
+            return
+        try:
+            os.kill(child, signal.SIGTERM)
+            os.waitpid(child, 0)
+        except OSError:
+            pass
+
+    # Check if a given field in GitHub is not as expected
+    def watches(self, token, watches):
+        for watch in watches:
+            try:
+                interval = int(watch.get('interval', None))
+            except TypeError:
+                interval = 60
```

Comment:
Then you will get a `ValueError`. IMHO failing explicitly with that is better than silently ignoring the bad configuration value and silently assuming 60.